### PR TITLE
documentation updated

### PR DESCRIPTION
**Pull request by filipgermanek**

### CODE REVIEW

**Comment by filipgermanek** [comment]
[Update API with connected projects](https://app.forecast.it/project/P-10/workflow/C-2367)

File: sections/allocations.md [hunk 1]
```diff
@@ -104,25 +110,27 @@
...
-|saturday | Integer, allocation hours in minutes (Defaults to persons working hours)|
-|sunday | Integer, allocation hours in minutes (Defaults to persons working hours)|
-|notes | String|
-
-\* Either a project or non_project_time must be supplied
+| Request fields    | Description/format                                                       |
+| ----------------- | ------------------------------------------------------------------------ |
+| project           | (Required\*) Integer, ID of project                                      |
+| non_project_time  | (Required\*) Integer, ID of non project time                             |
+| connected_project | Integer, ID of connected project                                         |
```
Comment (danny-larsen) [inline sections/allocations.md]: This must also be (Required*) like the others

File: sections/workflow_columns.md [hunk 2]
```diff
@@ -81,17 +150,40 @@ POST https://api.forecast.it/api/v1/projects/1/workflow_columns
...
+}
+```
+
+## Update project workflow column
+
+* `PUT /projects/{projectId}/workflow_columns/{workflow_columnId}` - Updates project workflow column. Returns the same object as getting a single workflow column.
+
+| Request fields | Description/format                                 |
+| -------------- | -------------------------------------------------- |
+| name           | (Required) String                                  |
```
Comment (danny-larsen) [inline sections/workflow_columns.md]: Not required on update

File: sections/workflow_columns.md [hunk 3]
```diff
@@ -100,9 +192,38 @@ PUT https://api.forecast.it/api/v1/projects/1/workflow_columns/1
...
 
-## Delete workflow column
+## Update connected project workflow column
 
-* `DELETE /projects/{projectId}/workflow_columns/{workflow_columnId}` - Deletes a workflow column. Does not delete cards in the workflow column.
+* `PUT /connected_projects/{connectedProjectId}/workflow_columns/{workflow_columnId}` - Updates connected project workflow column. Returns the same object as getting a single workflow column.
+
+| Request fields | Description/format                                 |
+| -------------- | -------------------------------------------------- |
+| name           | (Required) String                                  |
```
Comment (danny-larsen) [inline sections/workflow_columns.md]: Not required on update

File: sections/projects.md [hunk 4]
```diff
@@ -120,24 +128,26 @@
...
-|rate_card | Integer, ID of rate card|
+| Request fields               | Description/format                                              |
+| ---------------------------- | --------------------------------------------------------------- |
+| name                         | String                                                          |
+| stage                        | String (PLANNING, RUNNING, HALTED, DONE) (Defaults to PLANNING) |
+| status                       | String (GREEN, YELLOW, RED) (Defaults to GREEN)                 |
+| status_description           | String                                                          |
+| description                  | String                                                          |
+| estimation_units             | String (HOURS, POINTS) (Defaults to HOURS)                      |
+| minutes_per_estimation_point | Integer                                                         |
```
Comment (danny-larsen) [inline sections/projects.md]: Write what the default is set to

File: sections/connected_projects.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+
+| Request fields               | Description/format     |
+| ---------------------------- | ---------------------- |
+| name                         | String                 |
+| color                        | String                 |
+| estimation_units             | String (HOURS, POINTS) |
+| use_sprints                  | Boolean                |
+| sprint_length                | Integer                |
+| projects                     | Array, IDs of projects |
+| minutes_per_estimation_point | Integer                |
```
Comment (danny-larsen) [inline sections/connected_projects.md]: Add defaults to the fields that make sense. Estimation units, use sprints, sprint length and minutes per estimation point

**Pull request merged**